lttnganalyses/cli/mi.py

```diff
@@ -281,7 +242,48 @@ def _eq(self, other):
         return self._value == other._value
 
 
-class _SimpleName(_StructuredDataObject):
+class Boolean(_SimpleValue):
+    CLASS = 'bool'
+
+
+NEG_INF = '-inf'
+POS_INF = '+inf'
+
+
+class Number(_SimpleValue):
+    CLASS = 'number'
+
+    def __init__(self, value, low=None, high=None):
+        super().__init__(value)
+        self._low = low
```

Comment:
Is there any specific reason why `value` is publicly accessible, but not `_low` or  `_high`?